### PR TITLE
Add dynamic currency symbol and code option in Ecommerce

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:**  This PR adds implement for dynamic currency symbol and code option in Ecommerce

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-3616

**Merge checklist:**

- [ ] All reviewers approved
- [ ] Commits are (reasonably) squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable change.
Screenshot 2021-09-02 at 6 19 40 PM

### CODE REVIEW

File: ecommerce/settings/production.py [hunk 1]
```diff
@@ -140,6 +140,8 @@ def get_env_setting(setting):
 
 # Edly marketing site configuration
 EDLY_WORDPRESS_URL = config_from_yaml.get('EDLY_WORDPRESS_URL', EDLY_WORDPRESS_URL)
+OSCAR_DEFAULT_CURRENCY = 'USD'
```
Comment (zubair-arbi) [inline ecommerce/settings/production.py:143]: Please get prod values from yaml:

```python
config_from_yaml.get('OSCAR_DEFAULT_CURRENCY', OSCAR_DEFAULT_CURRENCY)
config_from_yaml.get('OSCAR_DEFAULT_CURRENCY_SYMBOL', OSCAR_DEFAULT_CURRENCY_SYMBOL)
```

**Review by burhandodhy — APPROVED** [review]
LGTM

**Review by zubair-arbi — CHANGES_REQUESTED** [review]
Just one suggestion, otherwise changes looks good.

File: ecommerce/settings/production.py (hunk 1 above)
Comment (asadmanzoor93) [reply ecommerce/settings/production.py]: Done

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Pull request merged**